src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -1281,14 +1283,18 @@ public OpenOfficePreferences getOpenOfficePreferences() {
                 getBoolean(OO_USE_ALL_OPEN_BASES),
                 getBoolean(OO_SYNC_WHEN_CITING),
                 getStringList(OO_EXTERNAL_STYLE_FILES),
-                get(OO_BIBLIOGRAPHY_STYLE_FILE));
+                get(OO_BIBLIOGRAPHY_STYLE_FILE),
+                get(OO_BIBLIOGRAPHY_STYLE_TYPE),
```

Comment:
For the enum use StyleType.valueOf(get(OO_BIBLIOGRAPHY_STYLE_TYPE))

as a related example for dealing with enums see GroupsPreferences